### PR TITLE
Feature/create angular project

### PR DESCRIPTION
**Pull request by jbarbaso**
Created angular project with ng new.
Removed tests files.
Added Bootstrap and NgBootstrap dependencies.
Added some project files.
Created a basic cars mock and displayed on screen as a table with simple search field.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "tslib": "^1.9.0",
+    "zone.js": "~0.8.26"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "~0.13.0",
+    "@angular/cli": "~7.3.8",
+    "@angular/compiler-cli": "~7.2.0",
+    "@angular/language-service": "~7.2.0",
+    "@types/node": "~8.9.4",
+    "@types/jasmine": "~2.8.8",
```
Comment (javieraviles) [inline package.json]: jasmine types?

Comment (jbarbaso) [reply package.json]: Removed!

File: src/app/cars/car-detail/car-detail.component.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<div *ngIf="car">
```
Comment (javieraviles) [inline src/app/cars/car-detail/car-detail.component.html]: I guess this ngIf is to avoid having a null in car while is loading which could throw errors. It would be better to do an optional evaluation (i.e. car?.id)

File: src/app/cars/entities/car.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export class Car {
+    id: number;
```
Comment (javieraviles) [inline src/app/cars/entities/car.ts]: id should be optional, not always Car model will contain the id and will still be consider as a Car (id?: number;)

File: src/test.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+// This file is required by karma.conf.js and loads recursively all the .spec and framework files
```
Comment (javieraviles) [inline src/test.ts]: is this file needed?

**Review by javieraviles — CHANGES_REQUESTED** [review]
some changes needed

File: src/app/cars/car-detail/car-detail.component.html (hunk 2 above)
Comment (jbarbaso) [reply src/app/cars/car-detail/car-detail.component.html]: Changed to an optional evaluation

File: src/app/cars/entities/car.ts (hunk 3 above)
Comment (jbarbaso) [reply src/app/cars/entities/car.ts]: Changed id to optional

File: src/test.ts (hunk 4 above)
Comment (jbarbaso) [reply src/test.ts]: Removed test.ts file

**Comment by jbarbaso** [comment]
Changes done
